### ISSUE TITLE
need to refresh printable calendar

### ISSUE THREAD

**Issue by brianblain** [jazzto/2015#53]
there was a change at Horseshoe - it's updated everywhere but not printable

**Comment by andrewfn** [comment]
I'll do that. The printable is a manual operation

**Comment by brianblain** [comment]
Thanks. i have your tix for sat night B

On Wed, Jun 24, 2015 at 8:31 PM, Andrew Fountain notifications@github.com
wrote:

> I'll do that. The printable is a manual operation
> On Jun 24, 2015 8:25 PM, "Brian Blain" notifications@github.com wrote:
> 
> > there was a change at Horseshoe - it's updated everywhere but not
> > printable
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/jazzto/2015/issues/53.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jazzto/2015/issues/53#issuecomment-115054395.

## 

Brian Blain
Managing Editor
Toronto Downtown Jazz
82 Bleecker Street, Toronto, ON  M4X 1L8
blain@tojazz.com

www.torontojazz.com
TD Toronto Jazz Festival
June 18-27, 2015

**Comment by andrewfn** [comment]
I rebuilt the calendar, and make sure that it was updated in ever layer of caching. We had over 2 million hits on the site (including images) so we are hiding behind a wall of heavy caching to keep the speed up.
B.t.w. stats are really up from last year. 42% higher on the opening night, and 20%-30% up for most of the time.
I'm really looking forward to Sat night. I may drop by on Friday if you are around.
